docs/faq/faq.md

```diff
@@ -40,3 +40,15 @@ There are some things you should consider before doing so:
   conflicts.
 
 TODO: Link to adoption use case once it gets on board.
+
+## Why Cluster API doesn't have the CRD YAML file?
+
+Kubernetes provides two methods to extend APIs: Custom Resource and Kubernetes API Aggregation Layer. For more detail, you can refer to [Extending the Kubernetes API](https://kubernetes.io/docs/concepts/extend-kubernetes/).
+
+Karmada uses the both extension methods. For example, `PropagationPolicy` and `ResourceBinding` use the `Custom Resource`, and `Cluster` resource uses the `Kubernetes API Aggregation Layer`.
+
+Therefore, `Cluster` resources do not have a CRD YAML file, and they are not visible when you execute the `kubectl get crd` command.
+
+So, why do we use the `Kubernetes API aggregation layer` to extend `Cluster` resources instead of using `custom resources`?
```

Comment:
```suggestion
So, why would you choose to use the Kubernetes API Aggregation Layer to extend `Cluster` resources instead of using Custom Resources?
```